changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+* [4254](https://github.com/zeta-chain/node/pull/4234) - add additional support for zetaclient public DNS name
```

Comment:
should we add to zetaclient config section in changelog that this is new option?